conf/routes

```diff
@@ -176,7 +176,7 @@ POST   /v3/topics/:guid/sendNotification @org.sagebionetworks.bridge.play.contro
 
 # Bridge Exporter
 POST /v3/recordexportstatuses      @org.sagebionetworks.bridge.play.controllers.HealthDataController.updateRecordsStatus
-POST /v3/recordExportStatuses      @org.sagebionetworks.bridge.play.controllers.HealthDataController.updateRecordsStatus
```

Comment:
Have we confirmed that recordExportStatuses is no longer being used? We should verify that it's being used before we delete it. Also, does this need to be included in this change?